apps/src/lab2/views/components/guide/Adlib.tsx

```diff
@@ -41,6 +41,16 @@ const Adlib: React.FunctionComponent<AdlibProps> = ({
     return output;
   }, [adlibOptions, options, template]);
 
+  // Compute joined choices text.
+  const joinedChoicesText = useMemo(() => {
```

Comment:
Optional/minor organization thing, I wonder if it would be cleaner for the `Adlib` to always just pass back the `string[]` array of selected choices, and have the calling component decide what to do with it (either combine it into the template, or join them, or something else)?